scripts/test_emscripten.sh

```diff
@@ -40,3 +40,5 @@ VERSION=$("$REPO_ROOT"/scripts/get_version.sh)
 
 echo "Running solcjs tests...."
 "$REPO_ROOT/test/externalTests/solc-js/solc-js.sh" "$SOLJSON" "$VERSION"
+echo "Running external tests...."
+"$REPO_ROOT/test/externalTests.sh" "$SOLJSON"
```

Comment:
Why is this needed?

Reply:
Not sure, will remove

Reply:
Probably conflict resolving mistake, removed.